README.md

```diff
@@ -188,6 +149,48 @@ fastify.listen(3000, err => {
 })
 ```
 
+### Custom error handler:
+
+You can optionally provide a custom errorHandler that will be used to handle any cleaning up: 
+
+```js
+'use strict'
+
+const fastify = require('fastify')()
+
+fastify.register(require('fastify-websocket'), {
+  errorHandler: function (error, conn /* SocketStream */, req /* FastifyRequest */, reply /* FastifyReply */) {
+    // Do stuff
+    // destroy/close connection
+    conn.destroy(error)
+  },
+  options: {
+    maxPayload: 1048576, // we set the maximum allowed messages size to 1 MiB (1024 bytes * 1024 bytes)
+    path: '/fastify', // we accept only connections matching this path e.g.: ws://localhost:3000/fastify
```

Comment:
Does the path option still make sense? We already have the router.

Reply:
You're right, it doesn't.
I'll update the readme file to advise against using the `path` option.
I also think it would be best to delete it (if provided) from the `options` object before passing it to the websocket server, while logging something to warn the user that it shouldn't be used. 

Let me know if you think it should be handled differently.

Reply:
Logging something if this is passed through is ok. Let's drop it from the docs as well.